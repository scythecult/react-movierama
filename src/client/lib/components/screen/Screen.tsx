import styles from './styles.module.css';

export const Screen = () => {
  return <div className={styles.screen}></div>;
};
