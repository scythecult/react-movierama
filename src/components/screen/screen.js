import React from "react";

import classes from "./screen.module.css";

const Screen = () => {
  return <div className={classes.screen}></div>;
};

export { Screen };
