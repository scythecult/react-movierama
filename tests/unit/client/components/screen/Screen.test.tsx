import { render } from '@testing-library/react';
import { Screen } from '@/client/lib/components/screen/Screen';

describe('Screen', () => {
  test('should correspond default layout', () => {
    const result = render(<Screen />);

    expect(result.container).toMatchSnapshot();
  });
});
