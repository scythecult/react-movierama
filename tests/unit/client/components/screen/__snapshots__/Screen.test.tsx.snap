// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Screen > should correspond default layout 1`] = `
<div>
  <div
    class="_screen_a4f98b"
  />
</div>
`;
